config/newsletter.php

```diff
@@ -47,6 +47,7 @@
     'mailjet' => [
         'key' => env('MJ_APIKEY_PUBLIC'),
         'secret' => env('MJ_APIKEY_PRIVATE'),
+        'connection_timeout' => (float)env('MJ_CONNECTION_TIMEOUT', '20'),
```

Comment:
We don't need to cast the value as it is done implicitly. And we should not define a value defaully so I would set it to null.